projects/netgrif-components/src/lib/header/header-modes/edit-mode/edit-mode.component.scss

```
@@ -5,3 +5,59 @@
+    align-items: center;
+    justify-content: center;
+    margin: 0 0 0 4px;
+    padding: 0;
+    border: 0;
+    background: transparent;
+    color: inherit;
+    cursor: pointer;
+    line-height: 0;
+
+    &:focus-visible {
+        outline: 2px solid currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)